editor/store/reducer.js

```diff
@@ -1042,8 +1054,19 @@ export const blockListSettings = ( state = {}, action ) => {
 	return state;
 };
 
+export const autosave = ( state = null, action ) => {
+	const { post } = action;
```

Comment:
Minor: Why do we destructure `post`, but not `type` ? Why not both or neither?

Reply:
not sure why that wound up like that :( Removed the destruring to better match other reducers in https://github.com/WordPress/gutenberg/pull/6257/commits/6cd8729523a790f48901ffae30142c516cb082f4